app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/attributeStrategies.ts

```diff
@@ -1,82 +1,156 @@
+                return FrequencyFeatureAttribute.getInstance(attributeName)
+            }
+            case FeatureAttributeName.transmissionPower: {
+                return TransmissionPowerFeatureAttribute.getInstance(attributeName)
+            }
+            
+            default : {
+                return IgnoreFeatureAttribute.getInstance(attributeName)
+                
+            }
+         }
     }
 
-    isNumeric() : boolean {
-        return (this.subclass? this.subclass.isNumeric() : false)
+    ignore(): boolean {
+        return false
+    }
+    getFilterType(){
+        return 'none'
     }
 
-    isString() : boolean {
-        return (this.subclass? this.subclass.isString() : false)
+    abstract isNumeric() : boolean 
 
-    }
-    
+    abstract isString() : boolean 
     
+}
 
+class TimestampFeatureAttribute extends FeatureAttribute implements FeatureAttributeTypes {
+    private static instance: TimestampFeatureAttribute;
+
+    private constructor(attributeName: FeatureAttributeName) {
+        super(attributeName)
+    }
+
+    public static getInstance(attributeName: FeatureAttributeName): TimestampFeatureAttribute {
+        if (!TimestampFeatureAttribute.instance) {
+            TimestampFeatureAttribute.instance = new TimestampFeatureAttribute(attributeName);
+        }
+
+        return TimestampFeatureAttribute.instance;
+    }
+    isNumeric() : boolean {
+        return true
+    }
+    isString() : boolean{
+        return false
+    }
+
+    getFilterType(){
+        return 'timeRange'
+    }
+    
 }
 
-class FeatureAttributeTimestamp implements FeatureAttributeFunctions {
-    private static instance: FeatureAttributeTimestamp;
-
-    /**
-     * The Singleton's constructor should always be private to prevent direct
-     * construction calls with the `new` operator.
-     */
-    private constructor() {}
-
-    /**
-     * The static method that controls the access to the singleton instance.
-     *
-     * This implementation let you subclass the Singleton class while keeping
-     * just one instance of each subclass around.
-     */
-    public static getInstance(): FeatureAttributeTimestamp {
-        if (!FeatureAttributeTimestamp.instance) {
-            FeatureAttributeTimestamp.instance = new FeatureAttributeTimestamp();
+class FrequencyFeatureAttribute extends FeatureAttribute implements FeatureAttributeTypes {
+    private static instance: FrequencyFeatureAttribute;
+
+    private constructor(attributeName: FeatureAttributeName) {
+        super(attributeName)
+    }
+
+    public static getInstance(attributeName: FeatureAttributeName): TimestampFeatureAttribute {
+        if (!FrequencyFeatureAttribute.instance) {
+            FrequencyFeatureAttribute.instance = new FrequencyFeatureAttribute(attributeName);
         }
 
-        return FeatureAttributeTimestamp.instance;
+        return FrequencyFeatureAttribute.instance;
     }
     isNumeric() : boolean {
```

Comment:
In general I prefer nullary (no parameter) abstract methods that are just accessing things and not doing calculations to be TypeScript getters/setters instead of methods.

If you do this in the abstract base class:
`abstract get numeric(): boolean`

Then you can either implement it in the subclass as a getter 
`get numeric() { ... }`
or as a readonly public attribute
`readonly numeric = false`